### ISSUE TITLE
List of find-linguistic-information-style papers (section 2)

### ISSUE THREAD

**Issue by boknilev** [boknilev/nlp-analysis-methods#5]
- [A Structural Probe for Finding Syntax in Word Representations](https://nlp.stanford.edu/~johnhew/structural-probe.html)
- [Introducing Orthogonal Constraint in Structural Probes](https://arxiv.org/abs/2012.15228) - also related to individual dimensions 
- [Evaluating the Representational Hub of Language and Vision Models](https://arxiv.org/abs/1904.06038)
- [LINSPECTOR: Multilingual Probing Tasks for Word Representations](https://arxiv.org/abs/1903.09442)
- [Improving Lemmatization of Non-Standard Languages with Joint Learning](https://arxiv.org/abs/1903.06939)
- [Linguistic Knowledge and Transferability of Contextual Representations](https://arxiv.org/abs/1903.08855)
- [To Tune or Not to Tune? Adapting Pretrained Representations to Diverse Tasks](https://arxiv.org/abs/1903.05987) - both probing classifiers and mutual information measures
- [CONVOLUTIONAL NEURAL NETWORKS CAN LEARN DURATION FOR DETECTING PITCH ACCENTS AND LEXICAL STRESS](https://www.ims.uni-stuttgart.de/institut/mitarbeiter/stehwisa/publications/stehwien_essv19.pdf)
- [DISCOVERY OF NATURAL LANGUAGE CONCEPTS IN INDIVIDUAL UNITS OF CNNS](https://arxiv.org/abs/1902.07249)
- [BERT Rediscovers the Classical NLP Pipeline](https://arxiv.org/pdf/1905.05950.pdf)
- ACL 2019: What does BERT learn about the structure of language?
- NAACL 2019: [Understanding and Improving Hidden Representation for Neural Machine Translation](https://www.aclweb.org/anthology/N19-1046) - also improve NMT by regularization according to analysis 
- One Size Does Not Fit All: Comparing NMT Representations of Different Granularities
- Section 5 in this [thesis](https://lirias.kuleuven.be/retrieve/541042) on Memory Analysis of LSTM Language Models
- ACL 2019: [Correlating neural and symbolic representations of language](https://arxiv.org/pdf/1905.06401.pdf)
- ACL 2019: [Empirical Linguistic Study of Sentence Embeddings](https://pdfs.semanticscholar.org/27f0/079a997bb7e3cd94884f40228148f42af5b2.pdf) - English and Polish 
- BlackboxNLP 2018: [Language Models Learn POS First](https://www.aclweb.org/anthology/W18-5438)
- EMNLP 2019: [Revealing the Dark Secrets of BERT](https://arxiv.org/abs/1908.08593)
- EMNLP 2019: Do Neural NLP Models Know Numbers? Probing Numeracy in Embeddings
- EMNLP 2019: How well do NLI models capture verb veridicality
- EMNLP 2019: Investigating BERT’s Knowledge of Language: Five Analysis Methods with NPIs
- EMNLP 2019: RSA, ELMo, BERT, eye tracking: Higher-order Comparisons of Sentence Encoder Representations
- [What Should/Do/Can LSTMs Learn
When Parsing Auxiliary Verb Constructions?](https://arxiv.org/pdf/1907.07950.pdf)
- [Analyzing deep CNN-based utterance embeddings for acoustic model
 adaptation](https://arxiv.org/abs/1811.04708)
- [Understanding Multi-Head Attention in Abstractive Summarization](https://arxiv.org/abs/1911.03898) - heads do different things, and some are redundant. 
- [Attention Can Reflect Syntactic Structure (If You Let It)](https://arxiv.org/abs/2101.10927)
- EMNLP 2019: [Investigating Multilingual NMT Representations at Scale](https://arxiv.org/abs/1909.02197)
- [Understanding Cross-Lingual Syntactic Transfer in Multilingual Recurrent Neural Networks](https://arxiv.org/pdf/2003.14056.pdf)
- EMNLP 2019: [The Bottom-up Evolution of Representations in the Transformer: A Study with Machine Translation and Language Modeling Objectives](https://arxiv.org/abs/1909.01380)
- EMNLP 2019: [How Contextual are Contextualized Word Representations? Comparing the Geometry of BERT, ELMo, and GPT-2 Embeddings](https://arxiv.org/abs/1909.00512)
- EMNLP 2019: [ Investigating BERT's Knowledge of Language: Five Analysis Methods with
 NPIs](https://arxiv.org/abs/1909.02597)
- EMNLP 2019: [Do NLP Models Know Numbers? Probing Numeracy in Embeddings](https://arxiv.org/abs/1909.07940)
- [A Primer in BERTology: What we know about how BERT works](https://arxiv.org/abs/2002.12327)

- [What does a network layer hear? Analyzing hidden representations of end-to-end ASR through speech synthesis](https://arxiv.org/pdf/1911.01102.pdf)
- [An empirical analysis of information encoded in disentangled neural speaker representations](https://arxiv.org/pdf/2002.03520.pdf)

- [Further Boosting BERT-based Models by Duplicating Existing Layers: Some Intriguing Phenomena inside BERT](https://arxiv.org/pdf/2001.09309.pdf)
- [Intrinsic Evaluation of Grammatical Information within Word Embeddings](http://jaslli.org/files/proceedings/46_paclic33_postconf.pdf)
- [Analyzing analytical methods: The case of phonology in neural models of spoken language](https://arxiv.org/pdf/2004.07070.pdf)
- [How recurrent networks implement contextual processing in sentiment analysis](https://arxiv.org/abs/2004.08013)
- [Dissecting Lottery Ticket Transformers: Structural and Behavioral Study of Sparse Neural Machine Translation](https://arxiv.org/pdf/2009.13270.pdf)
- [Picking BERT's Brain: Probing for Linguistic Dependencies in Contextualized Embeddings Using Representational Similarity Analysis](https://arxiv.org/abs/2011.12073)

- [Compositional Explanations of Neurons](https://arxiv.org/abs/2006.14032)
- [Intrinsic Probing through Dimension Selection](https://arxiv.org/pdf/2010.02812.pdf)
- [Neural Natural Language Inference Models Partially Embed Theories of Lexical Entailment and Negation](https://www.aclweb.org/anthology/2020.blackboxnlp-1.16.pdf) - section 6.2 does interventions. Also has other methods. 

Limitations and alternative approaches 
- EMNLP 2019: [Designing and Interpreting Probes with Control Tasks](https://arxiv.org/pdf/1909.03368.pdf)
- [Information-Theoretic Probing with MDL](http://lena-voita.github.io/posts/mdl_probes.html)
- [Information-Theoretic Probing for Linguistic Structure](https://arxiv.org/abs/2004.03061)
- [An information theoretic view on selecting linguistic probes](https://www.aclweb.org/anthology/2020.emnlp-main.744.pdf)
- [Investigating Transferability in Pretrained Language Models](https://arxiv.org/abs/2004.14975)
- [Perturbed Masking: Parameter-free Probing for Analyzing and Interpreting BERT](https://arxiv.org/abs/2004.14786)
- [Asking without Telling: Exploring Latent Ontologies in Contextual Representations](https://arxiv.org/abs/2004.14513)
- [How to Probe Sentence Embeddings in Low-Resource Languages: On Structural Design Choices for Probing Task Evaluation](https://arxiv.org/abs/2006.09109)
- [Classifier Probes May Just Learn from Linear Context Features](https://www.aclweb.org/anthology/2020.coling-main.450.pdf)
